backend/accounts/feetarget.go

```diff
@@ -22,6 +22,7 @@ import (
 // Coin specific methods are implemented in corresponding coins.
 type FeeTarget interface {
 	Code() FeeTargetCode
+	FeeRateInfo() string
```

Comment:
"info" to me usually means some structured data. something like `FmtFeeRate` or `FormattedFeeRate` with a comment `// xxx returns a formatted fee rate to display in the UI` might have been closer to what it does.

I mean here and below in `jsonFeeTarget`.

Reply:
Done